src/navfeed.sol

```diff
@@ -133,7 +146,16 @@ contract NAVFeed is BaseNFTFeed, Interest, Buckets  {
         buckets[maturityDate_].value = safeAdd(buckets[maturityDate_].value, futureValue[nftID_]);
     }
 
-    function repay(uint loan, uint amount) external auth {
+    function repay(uint loan, uint amount) external auth returns (uint navDecrease) {
+        uint navDecrease = _repay(loan, amount);
+        if (navDecrease > approximatedNAV) {
+            approximatedNAV = 0;
+        }
+        approximatedNAV = safeSub(approximatedNAV, navDecrease);
+        return navDecrease;
```

Comment:
don't think we need to return anything here, would change the interface of the function